tests/aiohttp/test_aiohttp_app.py

```diff
@@ -26,7 +26,7 @@ def test_app_run(web_run_app_mock, aiohttp_api_spec_dir):
     app.run(use_default_access_log=True)
     logger = logging.getLogger('connexion.aiohttp_app')
     assert web_run_app_mock.call_args_list == [
-        mock.call(app.app, port=5001, host='0.0.0.0', access_log=logger)
+        mock.call(app.app, port=5001, host='0.0.0.0', access_log=logger, use_default_access_log=True)
```

Comment:
Why is this change needed here? Isn't the **options sufficient in aiohttp_app? or do tests break without this?

Reply:
I see the error. See my other comment for how I would prefer to make the tests pass.